docs/en_US/mnist_examples.md

```diff
@@ -1,28 +1,126 @@
+
+ ```bash
+    nnictl create --config nni/examples/trials/*mnist*/*config*.yml
+```
+
+## Examples
+
+The examples are listed below:
+
+
+|item name|search space|platform|framework|tuner|code directory
+|---|---|---|---|---|---|
+|[MNIST with NNI API](#mnist)|json|local,remote,pai,kubeflow|tensorflow|tpe|[examples/trials/mnist/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist)
+|[MNIST with NNI annotation](#mnist-annotation)|annotation|local,remote,pai,kubeflow|tensorflow|tpe|[examples/trials/mnist-annotation/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-annotation)
+|[MNIST in keras](#mnist-keras)|json|local,remote,pai|keras|tpe|[examples/trials/mnist-keras/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-keras)
+|[MNIST -- tuning with batch tuner](#mnist-batch)|json|local,pai,remote|keras|batch|[/examples/trials/mnist-batch-tune-keras](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-batch-tune-keras)
+|[MNIST -- tuning with hyperband](#mnist-hyperband)|json|local,pai,remote|tensorflow|hyperband|[examples/trials/mnist-hyperband/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-hyperband)
+|[MNIST -- tuning within a nested search space](#mnist-nested)|json|local,pai,remote|tensorflow|tpe|[examples/trials/mnist-cascading-search-space/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-cascading-search-space)
+|[distributed MNIST (tensorflow) using kubeflow](#mnist-kubeflow-tf)|json|local,remote,pai,kubeflow|tensorflow|tpe|[examples/trials/mnist-distributed/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-distributed)
+|[distributed MNIST (pytorch) using kubeflow](#mnist-kubeflow-pytorch)|json|local,remote,pai,kubeflow|pytorch|tpe|[examples/trials/mnist-distributed-pytorch/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-distributed-pytorch)
+
+
+
+<a name ="mnist"></a>
 **MNIST with NNI API**
 
-This is a simple network which has two convolutional layers, two pooling layers and a fully connected layer. We tune hyperparameters, such as dropout rate, convolution size, hidden size, etc. It can be tuned with most NNI built-in tuners, such as TPE, SMAC, Random. We also provide an exmaple YAML file which enables assessor.
+This is a simple network which has two convolutional layers, two pooling layers and a fully connected layer. We tune hyperparameters, such as dropout rate, convolution size, hidden size, etc. It can be tuned with most NNI built-in tuners, such as TPE, SMAC, Random. We also provide an exmaple yaml file which enables assessor.
+
+As we stated in the target, we target to find out the best `optimizer` for training . When using different optimizers, we also need to adjust `dropout_rate` ,`conv_size`,`hidden_size`,`batch_size`and `learning_rate` accordingly. so we chose these three parameters as hyperparameters and write the following search space.
```

Comment:
I think " we target to find out the best `optimizer` for training " belong to the "pytorch example".... I don't know why you add this sentence here...